lib/xlat_tables_v2/xlat_tables_utils.c

```diff
@@ -539,7 +539,9 @@ int xlat_change_mem_attributes_ctx(const xlat_ctx_t *ctx, uintptr_t base_va,
 		 * before writing the new descriptor.
 		 */
 		*entry = INVALID_DESC;
-
+#if !HW_ASSISTED_COHERENCY
+		dccvac((uintptr_t)entry);
```

Comment:
I note you're doing a clean here rather than a clean+invalidate like the other places. I think a clean is enough though.

Reply:
I've changed the rest.